core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -679,7 +679,7 @@ public void destroyCache(final String alias) throws CachePersistenceException {
     }
 
     try {
-      removeAndCloseWithoutNotice(alias);
+      removeCache(alias);
```

Comment:
Yes. But we are not testing that there are no notifications :-) Also, if in maintenance mode from destroyCache, it should stay there all the way through. Not sure this is tested as well.